index.js

```diff
@@ -69,6 +69,34 @@ class Executor {
         throw new Error('Not implemented');
     }
 
+    /**
+      * Starts a new periodic build in an executor
+      * @method _startPeriodic
+      * @param {Object} config               Configuration
+      * @return {Promise}
+      */
+    startPeriodic(config) {
+        return this._startPeriodic(config);
+    }
+
+    async _startPeriodic() {
```

Comment:
We should implement this function for all executors and make them resolve before the model one got merged. https://github.com/screwdriver-cd/models/pull/239/files

Otherwise, it will break for ppl who are not using executor-queue even they don't want to use build periodically.

Reply:
Relevant PRs:

https://github.com/screwdriver-cd/executor-jenkins/pull/19
https://github.com/screwdriver-cd/executor-k8s-vm/pull/32
https://github.com/screwdriver-cd/executor-k8s/pull/82